docs/_scripts/recursive_add_tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "Retrieve the href of Tag in the synced repository."
+export TAG_URL=$(http $BASE_ADDR$'/pulp/api/v3/content/docker/tags/?repository_version='$REPOVERSION_HREF'&name=latest' \
```

Comment:
any reason we have in this pr _URL and _HREF, is there any distinction between them?

Reply:
nope, i've switched from URL to HREF